### DISCUSSION TITLE
Quoted search doesnt return exact match of the phrase

### DISCUSSION THREAD

**Discussion by meghanajoshilw** [ravendb/ravendb#16688] [Q&A] [answered]
If you do a search for "red green" in quotes, the results come back with documents that have "red green" and also "red and green" in them. And we do not want the query/analyzer to include the word and. How do we manipulate it? 

We are using Standard analyzer and the Indexing type is Search. 

![image](https://github.com/ravendb/ravendb/assets/133284420/3cf2f14a-a4c5-46cc-8d29-7bbd9b6f4a29)

**Comment by ayende** [comment]
When using the standard analyzer, the 'and' word is being removed, so as far as we are concerned, that is the same thing.
You can use white space analyzer, instead, which will not remove stop words.

**Reply by meghanajoshilw** [reply]
Thank you Ayende for getting back. I think you missed it in my question where it said we did **not** want any stop words in the results. And we're already using the Standard Analyzer but we still find "and" in the results. 

For example, when we do a quoted search like "red green" - 
we get the results for
1.  red green 
2.  red and green

And we dont want the second result set. So the question was how do we eliminate that and only keep the first result set (the one which doesnt have the stop word in it)? 

Let me know if this makes sense. Thanks!

**Reply by ayende** [reply, marked as answer]
You need to use the `WhiteSpace Analyzer`, see: 
![image](https://github.com/ravendb/ravendb/assets/116915/fed51609-f28d-4941-af70-615f278f2f9e)

**Reply by meghanajoshilw** [reply]
Will try it out, thank you!

**Reply by meghanajoshilw** [reply]
Thank you very much! That worked. Appreciate the help.
